### ISSUE TITLE
Questions about Binary CE

### ISSUE THREAD

**Issue by DeepRunner** [itaigat/removing-bias-in-multi-modal-classifiers#3]
In the train.py, when computing cross entropy, it uses torch.nn.functional.binary_cross_entropy_with_logits:
   inf_output = model(inf_image, inf_question)
   inf_loss = torch.nn.functional.binary_cross_entropy_with_logits(inf_output, expanded_logits)

However, inf_output is an N-dim vector where N denotes the size of all possible answers. Why binary cross entropy used, rather than cross entropy?

Thank you!

**Comment by itaigat** [comment]
Hey, 

Each VQA answer is annotated with 10 annotators (you can find more information here - https://visualqa.org/). This result in 'soft' labels, for example, a possible labels are:
[1/3, 0, 0, 2/3] or [1, 0, 0, 0] or [1, 0, 1/3, 0]. 

Thus, to consider this information, we (and many others) use the binary cross-entropy loss rather than cross-entropy loss which is usually used in multi-class tasks.
